### PR TITLE
#166187848 Configuring DB ENGINE

### PR DESCRIPTION
**Pull request by Emmanuelaaron**
#### What does this PR do?

Configures the project to use postgreSQL as  DB ENGINE instead of using the default engine.

#### Description of task completed

Refactoring DATABASES dictionary in settings.py file under the default key to having PostgreSQL settings

#### How should this be manually tested
* Make sure you have [PostgreSQL](https://www.postgresql.org/download/macosx/) installed on your local machine
* Open a [PostgreSQL](https://www.postgresql.org/download/macosx/) terminal using `$ psql` on your local machine
* Do a `git fetch origin ch-db_connection-166187848`
* Checkout into `ch-db_connection-166187848`
* Install all the dependencies using `pip install -r requirements.txt`
* Create a `.env` file.
* Inside the .env file insert the code below
`export DB_NAME='authors'`
`export DB_USER=''`
`export DB_PASS=''`
`export DB_HOST='localhost'`
`export DB_PORT='5432'`
`echo 'env variables set' `
* Then `source .env` in your terminal
* Then migrate using `$ python manage.py migrate`

[mantains #166187848](https://www.pivotaltracker.com/n/projects/2350340/stories/166187848)
-

### CODE REVIEW

File: authors/apps/authentication/tests/test_authentication.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import json
+
+from django.contrib.auth.models import User
```
Comment (alexxsanya) [inline authors/apps/authentication/tests/test_authentication.py]: @Emmanuelaaron I notice User model imported but not used; I suggest we remove the unused imports.

File: authors/settings.py [hunk 2]
```diff
@@ -24,8 +24,6 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
```
Comment (jennizalwango) [inline authors/settings.py]: @Emmanuelaaron  l would like to know why did you remove line 27 in your settings .py, am l missing out on anything?

File: authors/settings.py [hunk 3]
```diff
@@ -24,8 +24,6 @@
...
-
```
Comment (jennizalwango) [inline authors/settings.py:28]: @Emmanuelaaron l would like to know why you removed line 27 in your settings.py that is `ALLOWED_HOSTS`
Am l missing out on anything

File: authors/apps/authentication/tests/test_authentication.py (hunk 1 above)
Comment (Emmanuelaaron) [reply authors/apps/authentication/tests/test_authentication.py]: Thank you @alexxsanya for taking note of that. I have resolved it.

File: authors/settings.py (hunk 3 above)
Comment (Emmanuelaaron) [reply authors/settings.py:28]: The line was removed because it had no purpose.
